tests/platforms/kubernetes/appmanager.go

```diff
@@ -187,6 +192,31 @@ func (m *AppManager) ValidiateSideCar() (bool, error) {
 	return true, nil
 }
 
+// ScaleDeploymentReplica scales the deployment
+func (m *AppManager) ScaleDeploymentReplica(replicas int32) error {
+	if replicas < 0 {
```

Comment:
Add an upper bound limit to avoid an error when callers scale out with an arbitrarily large number.

Reply:
Yeah I would add upper bound later. because yaron merged this pr already.

Reply:
OK.